src/analysis/normed_space/basic.lean

```diff
@@ -409,35 +407,67 @@ lemma squeeze_zero_norm {f : γ → α} {g : γ → ℝ} {t₀ : filter γ}
   tendsto f t₀ (𝓝 0) :=
 squeeze_zero_norm' (eventually_of_forall h) h'
 
-lemma lim_norm (x : α) : tendsto (λg : α, ∥g - x∥) (𝓝 x) (𝓝 0) :=
-tendsto_iff_norm_tendsto_zero.1 (continuous_iff_continuous_at.1 continuous_id x)
+lemma tendsto_norm_sub_self (x : α) : tendsto (λ g : α, ∥g - x∥) (𝓝 x) (𝓝 0) :=
+by simpa [dist_eq_norm] using tendsto_id.dist (tendsto_const_nhds : tendsto (λ g, (x:α)) (𝓝 x) _)
 
-lemma lim_norm_zero : tendsto (λg : α, ∥g∥) (𝓝 0) (𝓝 0) :=
-by simpa using lim_norm (0:α)
+lemma tendsto_norm {x : α} : tendsto (λg : α, ∥g∥) (𝓝 x) (𝓝 ∥x∥) :=
+by simpa using tendsto_id.dist (tendsto_const_nhds : tendsto (λ g, (0:α)) _ _)
 
-lemma continuous_norm : continuous (λg:α, ∥g∥) :=
-begin
-  rw continuous_iff_continuous_at,
-  intro x,
-  rw [continuous_at, tendsto_iff_dist_tendsto_zero],
-  exact squeeze_zero (λ t, abs_nonneg _) (λ t, abs_norm_sub_norm_le _ _) (lim_norm x)
-end
+lemma tendsto_norm_zero : tendsto (λg : α, ∥g∥) (𝓝 0) (𝓝 0) :=
+by simpa using (tendsto_norm : tendsto _ (𝓝 (0:α)) _)
```

Comment:
```suggestion
by simpa using tendsto_norm_sub_self (0:α)
```
This was the original proof of the lemma, I had changed it as an experiment but the change isn't actually better.